src/kixi/ckan.clj

```diff
@@ -157,24 +157,23 @@
          (throw t)))))
 
   (-datastore-upsert [this id data]
-    (let [url      (str "https://"(-> this :ckan-client-session :site)
+    (let [url      (str "http://" (-> this :ckan-client-session :site)
```

Comment:
why no SSL? I think we probably need a function that creates this for us that we can configure. For now a function (w/o config) should work. wdyt?